steps/cadence-innovus-flowsetup/innovus-foundation-flow/custom-scripts/always-source.tcl

```diff
@@ -20,3 +20,16 @@ source $vars(adk_dir)/adk.tcl
 
 mkdir -p $vars(rpt_dir)
 
+# [08/2021] There seems to be a source of nondeterminism when
+# building e.g. garnet glb tile. Also see garnet issue
+# https://github.com/StanfordAHA/garnet/issues/803
+# 
+# 
+#   **WARN: (IMPECO-560): The netlist is not unique, because the module
+#   'Tile_PE_mux_logic_1_20' is instantiated multiple times. Make the
+#   netlist unique by running 'set init_design_uniquify 1' before
+#   loading the design to avoid the problem.
+#
+# So...
+
+set init_design_uniquify 1
```

Comment:
Okay.. I think it is not a bad idea to default to uniquifying the design. However, it should not be here.

Can you put it in cadence-innovus-init/scripts/main.tcl just before the source? Also, we don't need to put a comment to explain why it is there.

Reply:
Ah yes, that's definitely better. Done. Thanks for the feedback!